src/app/chart/price-chart-builder.ts

```diff
@@ -63,21 +81,37 @@ export class PriceChartBuilder {
       this.log.error('Chart not yet init');
     }
     const data = [];
+    const volumeData = [];
     dtos?.forEach(dto => {
       if (dto.timestamp > this.lastDate) {
         this.lastDate = dto.timestamp;
       }
       this.lastOhlc = dto;
+      volumeData.push({time: dto.timestamp, value: dto.volume});
       data.push({time: dto.timestamp, open: dto.open, high: dto.high, low: dto.low, close: dto.close});
     });
+    this.volumeSeries = this.chart.addHistogramSeries({
```

Comment:
check `if (!this.volumeSeries) {` otherwise you will create a ton of series on the chart